apps/framework-cli/src/cli/routines.rs

```diff
@@ -666,25 +674,73 @@ pub async fn remote_plan(
 }
 
 fn spawn_connection_monitor(redis_client: Arc<Mutex<RedisClient>>) {
+    // Create a watch channel for signaling shutdown
```

Comment:
Why do we need all this if we trigger the shutdown in the webserver?

Reply:
It was a way of dealing with the panics that resulting during shutdowns.  Seems that panics can sneak through the cracks.  I confirmed we no longer need the monitor